ProjectSettings/ProjectSettings.asset

```diff
@@ -267,7 +273,7 @@ PlayerSettings:
   AndroidTargetDevices: 0
   AndroidSplashScreenScale: 0
   androidSplashScreen: {fileID: 0}
-  AndroidKeystoreName: '{inproject}: Keystore/miragexr.keystore'
```

Comment:
might have to be removed? I think we did not keep it in develop, only set it for releases, right?

Reply:
Yes, we don't store the key in the repository and it is used only for the release